### ISSUE TITLE
Option 'token' is deprecated. Use 'apiKey' instead.

### ISSUE THREAD

**Issue by MadSandwich** [reportportal/agent-js-webdriverio#51]
While using token as described in the configuration example -> https://github.com/reportportal/examples-js/blob/main/example-webdriverio/examples-webdriverio-mocha/wdio.conf.js
Everything is working but receiving annoying warn message to use apiKey instead, whenever I set apiKey instead of token I receive an error:
```js
ReportPortalRequiredOptionError: 
ReportPortal client error: 
Validation failed. Please, check the specified parameters: 
Property 'apiKey' must not be empty.
    at getApiKey (path/node_modules/@reportportal/client-javascript/lib/commons/config.js:25:13)
    at getClientConfig (path/node_modules/@reportportal/client-javascript/lib/commons/config.js:40:20)
    at new RPClient (path/node_modules/@reportportal/client-javascript/lib/report-portal-client.js:34:19)
    at new Reporter (path/node_modules/@reportportal/agent-js-webdriverio/src/reporter.ts:57:19)
    at BaseReporter._loadReporter (file://path/node_modules/@wdio/runner/build/reporter.js:161:20)
    at Array.map (<anonymous>)
    at BaseReporter.initReporters (file://path/node_modules/@wdio/runner/build/reporter.js:23:68)
    at Runner.run (file://path/node_modules/@wdio/runner/build/index.js:70:30)
```

wdio.config
```js
const rpConfig = {
  apiKey: 'my_token',
  endpoint: 'url/api/v1',
  project: 'xxx',
  launch: `Launch name`,
  mode: 'DEFAULT',
  debug: false,
  attributes: [{ key: 'Platform', value: process.env.ENV }],
  attachPicturesToLogs: true,
  reportSeleniumCommands: false,
  seleniumCommandsLogLevel: 'info',
};
```

**Comment by owens-ben** [comment]
Having the same issue.

**Comment by AmsterGet** [comment]
Hi guys!
Thanks for pointing that.
Fix is on the way!
So until the release please use the `token` option as a workaround.

**Comment by DzmitryHumianiuk** [comment]
@MadSandwich, could you clarify the problem?

With version 23.2, ReportPortal (RP) transitioned from using tokens to API keys. Previously, tokens had a one-to-one relationship with each account. Now, API keys can support many-to-one relationships (multiple keys linked to a single account).

For older versions, you can still enter the API key into the token field. However, the newer versions of the JavaScript (JS) agents require an API key field in the configuration instead of a token field.

It's likely you can continue using the token field for now, but you'll be prompted to switch to an API key eventually.

For compatibility with the latest version of RP, we suggest updating to the newest version of the RP JavaScript libraries.

Please let me know if my understanding of your issue is incorrect.

**Comment by AmsterGet** [comment]
Fixed in version [5.1.0](https://github.com/reportportal/agent-js-webdriverio/releases/tag/v5.1.0).
